integration/test/test_epoch_inference.py

```diff
@@ -133,14 +133,19 @@ def test_trace_epoch_count(self):
         """
         for run_config in self._config_names:
             report_path = 'test_{}_{}.report'.format(self._test_name, run_config)
-            trace_path = '{}_{}'.format(self._trace_path_prefix, run_config)
-            output = geopmpy.io.AppOutput(report_path, trace_path+'*')
-            node_list = output.get_node_names()
+            raw_report = geopmpy.io.RawReport(report_path)
+            node_list = raw_report.host_names()
+
+            profile_trace_path = '{}_profile_{}'.format(self._trace_path_prefix, run_config)
+            output = geopmpy.io.AppOutput(profile_trace_path+'*')
```

Comment:
```suggestion
            output = geopmpy.io.AppOutput(f'{profile_trace_path}*')
```

Reply:
I'm a little confused about how AppOutput is only being passed the glob for the profile trace files.  I kind of expected this to be a parameterized or somehow differenciated.